### PR TITLE
:rewind: Add ability to override current time

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Comment by st3v3nhunt** [comment]
I'll look at this tomorrow

**Comment by neilbmclaughlin** [comment]
@st3v3nhunt - ready for review/merge

**Comment by st3v3nhunt** [comment]
Functionality it looks ok apart from the date displayed in the footer is showing as 01:25 rather than 13:25 as would have expected

**Comment by st3v3nhunt** [comment]
I like the cheeky 'setting it for everybody rather than making it something each client can control' approach 😜  
Very prototype-y

File: app/lib/dateUtils.js [hunk 1]
```diff
@@ -38,7 +51,7 @@ function timeInRange(date, open, close) {
     }
   }
 
-  console.log([date.format(), start.format(), end.format()]);
+  // console.log([date.format(), start.format(), end.format()]);
```
Comment (st3v3nhunt) [inline app/lib/dateUtils.js]: Remove this at some point?

**Pull request merged**